module/support/logging/DataPlayback/README.md

```diff
@@ -2,13 +2,20 @@ DataPlayback
 ============
 
 ## Description
+DataPlayback is a module which will allow you to play back `.nbs` files as if they were happening in realtime.
+This can be very useful for when you are trying to diagnose a problem for which you have a recording as you can continuously play back the same data.
 
+The module will buffer up packets from the future and emit them at the same rate that they were emitted originally.
 
 ## Usage
+In order to play back the messages, you need to create a role with this module that has the modules you wish to test. You then need to enable the message types that are the input to these modules by setting them to true in the configuration (e.g. `message.input.sensors: true`). Make sure the `.nbs` file contains the messages you are trying to emit (you can check with `./b nbs stats file`).
```

Comment:
```suggestion
In order to play back the messages, you need to create a role with this module that has the modules you wish to test. You then need to enable the message types that are the input to these modules by setting them to true in the configuration (e.g. `message.input.Sensors: true`). Make sure the `.nbs` file contains the messages you are trying to emit (you can check with `./b nbs stats file`).
```

Reply:
You also need to make sure that any modules that emit a message you are playing back are removed from the role (e.g. is `message.input.Sensors` is being played back `platform::darwin::SensorFilter` and/or `platform::darwin::HardwareIO`/`platform::darwin::HardwareSimulator` should **not** be in the role as it could create conflicting messages in the system).